dmfriends/urls.py

```diff
@@ -13,7 +13,8 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
+    path('', include('product.urls'))
```

Comment:
그냥 여기에 product를 주시는걸 권장 드립니다.
```suggestion
    path('product', include('product.urls'))
```